src/conversion/configurationdefinedconverters.js

```diff
@@ -0,0 +1,310 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module engine/conversion/configurationdefinedconverters
+ */
+
+import AttributeElement from '../view/attributeelement';
+import ViewContainerElement from '../view/containerelement';
+
+import buildModelConverter from './buildmodelconverter';
+import buildViewConverter from './buildviewconverter';
+
+/**
+ * Helper for creating model to view converter from model's element
+ * to {@link module:engine/view/containerelement~ContainerElement ViewContainerElement}. The `acceptAlso` property is ignored.
+ *
+ * You can define conversion as simple model element to view element conversion using simplified definition:
+ *
+ *		modelElementToViewContainerElement( {
+ *			model: 'heading1',
+ *			view: 'h1',
+ *		}, [ dispatcher ] );
+ *
+ * Or defining full-flavored view object:
+ *
+ *		modelElementToViewContainerElement( {
+ *			model: 'heading1',
+ *			view: {
+ *				name: 'h1',
+ *				class: [ 'header', 'article-header' ],
+ *				attributes: {
+ *					data-header: 'level-1',
+ *				}
+ *			},
+ *		}, [ dispatcher ] );
+ *
+ * Above will generate an HTML tag:
+ *
+ *		<h1 class="header article-header" data-header="level-1">...</h1>
+ *
+ * @param {module:engine/conversion/configurationdefinedconverters~ConverterDefinition} definition A conversion configuration.
+ * @param {Array.<module:engine/conversion/modelconversiondispatcher~ModelConversionDispatcher>} dispatchers
+ */
+export function modelElementToViewContainerElement( definition, dispatchers ) {
+	const { model: modelElement, viewDefinition } = parseConverterDefinition( definition );
+
+	buildModelConverter()
+		.for( ...dispatchers )
+		.fromElement( modelElement )
+		.toElement( () => ViewContainerElement.fromViewDefinition( viewDefinition ) );
+}
+
+/**
+ * Helper for creating view to model converter from view to model element. It will convert also all matched view elements defined in
+ * `acceptAlso` property. The `model` property is used as model element name.
+ *
+ * Conversion from model to view might be defined as simple one to one conversion:
+ *
+ *		viewToModelElement( { model: 'heading1', view: 'h1' }, [ dispatcher ] );
+ *
+ * As a full-flavored definition:
+ *
+ *		viewToModelElement( {
+ *			model: 'heading1',
+ *			view: {
+ *				name: 'p',
+ *				attributes: {
+ *					'data-heading': 'true'
+ *				},
+ *				// it might require to define higher priority for elements matched by other features
+ *				priority: 'high'
+ *			}
+  *		}, [ dispatcher ] );
+ *
+ * or with `acceptAlso` property to match many elements:
+ *
+ *		viewToModelElement( {
+ *			model: 'heading1',
+ *			view: {
+ *				name: 'h1'
+ *			},
+ *			acceptAlso: [
```

Comment:
Maybe `view` could be an `array` in case of multiple definitions?

Reply:
The `view` option defines default model->view conversion and it should always be one as it would be impossible to convert one model value to many (random?) view ;).

The `acceptAlso` is used as a way to convert existing markup to model in view->model conversion. As in this example you always want to convert to `h1` element but when loading content you want to map other elements to `heading1` also.

So TL;DR: I don't think so ;)

Reply:
I mean view as array only for view -> model conversion. I wouldn't be afraid that someone will try to use it for converting to multiple random element in case of model -> view conversion.

Reply:
Or I don't understand something. Maybe exactly the same definition will be used for both conversions.

Reply:
Bu the configuration (like in heading) would be one: 

```js
return VirtualTestEditor
	.create( {
		plugins: [ HeadingEngine ],
		heading: {
			options: [
				{ model: 'paragraph', title: 'paragraph' },
				{
					model: 'heading1',
					view: 'h1',
					acceptsAlso: [
						{ name: 'p', attributes: { 'data-heading': 'h1' }, priority: 'high' }
					],
					title: 'User H1'
				}
			]
		}
	} )
`

Reply:
Ok, makes sense now :)

Reply:
And also the feature would use it as:

https://github.com/ckeditor/ckeditor5-heading/blob/b8ef80f86121f8a1ec20c66314bf09eb837cb818/src/headingengine.js#L85-L100